doc/man7/provider-signature.pod

```diff
@@ -414,6 +414,11 @@ the error and set the approved "fips-indicator" to 0.
 This option is used by the OpenSSL FIPS provider, and breaks FIPS compliance if
 set to 0.
 
+=item "verify-digested" (B<OSSL_SIGNATURE_PARAM_FIPS_VERIFY_DIGESTED> <integer>
+
```

Comment:
Please specify when this should be set here..

Reply:
They should probably also state that they break FIPS compliance..

Reply:
should this be "no-verify-digested"? You have "no-short-mac" for example..

Reply:
Yes, it needed the `no-` prefix.
Added descriptive text.